File: src/com/eagle/programmar/Perl/Terminals/Perl_KeywordChoice.java
// Copyright Eagle Legacy Modernization LLC, 2010-date
// Original author: Steven A. O'Hara, Nov 17, 2013

package com.eagle.programmar.Perl.Terminals;

import com.eagle.tokens.TerminalKeywordChoice;

public class Perl_KeywordChoice extends TerminalKeywordChoice
{
	// Need default constructor for reading from the XML file
	public Perl_KeywordChoice()
	{
		super();
	}

	public Perl_KeywordChoice(String... words)
	{
		super(words);
	}
}

File: csrc/com/eagle/programmar/Perl/Terminals/Perl_KeywordChoice.cs
// ====================================================================================================
// Produced by the Free Edition of Java to C# Converter.
// Purchase a Premium Edition license at:
// https://www.tangiblesoftwaresolutions.com/order/order-java-to-csharp.html
// ====================================================================================================

// Copyright Eagle Legacy Modernization LLC, 2010-date
// Original author: Steven A. O'Hara, Nov 17, 2013

namespace com.eagle.programmar.Perl.Terminals
{
	using TerminalKeywordChoice = com.eagle.tokens.terminals.TerminalKeywordChoice;

	public class Perl_KeywordChoice : TerminalKeywordChoice
	{
		// Need default constructor for reading from the XML file
		public Perl_KeywordChoice() : base()
		{
		}

		public Perl_KeywordChoice(params string[] words) : base(words)
		{
		}
	}
}
